### DISCUSSION TITLE
Should we add hermes version to all CLIs?

### DISCUSSION THREAD

**Discussion by ancazamfir** [informalsystems/hermes#2803] [General]
The question of "what hermes version are you running?" often pops up when triaging issues. Although the information is required in the github template there is no enforcement. Also sometimes this happens over discord or other channels and there is no github issue yet, we get dump pieces from hermes output.

**Comment by adizere** [comment]
Would this mean that any time a CLI runs, as part of log output we would publish the Hermes `v.x.y.z` ? Great idea!

@seanchen1991 what do you think? It's unclear how we can ensure we can _always_ print the version, even in corner-cases eg when there's an `absissa` framework error (inability to parse config file, for example). We might need to investigate a bit to understand the design constraints.

**Comment by seanchen1991** [comment]
To me, this sounds like a fair bit of work for not a lot of benefit. If the idea is that we eliminate the need to ask users what Hermes version they're running, I don't see that as much of a gain.

I understand that it can be tedious to always ask users for their Hermes version and their config details, but ultimately we'll never be able to automate away the necessity of interfacing with the user.

**Comment by ljoss17** [comment]
It seems like adding the version to most of the CLI outputs only implies very few changes. Here is a PR to show a possible solution https://github.com/informalsystems/hermes/pull/2883.
What do you think about this solution ?
